### ISSUE TITLE
Document how to integrate the framework as Embed Framework

### ISSUE THREAD

**Issue by davidandreoletti** [swisspol/GCDWebServer#237]
I use Carthage to fetch/build the framework (v3.2.7) in my Swift enabled Xcode project. 

The integration process followed is:
- Drag GCDWebServer.framework to Embedded Binaries in the project's Build Settings
- Add #import <GCDWebServers/GCDWebServers.h> to myapp-Bridging-Header.h

Compilation is successfull. However, running this code return a nil webuploader:
````
import GCDWebServers
....

let paths = NSSearchPathForDirectoriesInDomains(.DocumentDirectory, .UserDomainMask, true);
let documentsPath = paths.first;
var webuploader = GCDWebUploader(uploadDirectory: documentsPath!);
````
webuploader is nil because [GCDWebUploader.m's initWithUploadDirectory | https://github.com/swisspol/GCDWebServer/blob/master/GCDWebUploader/GCDWebUploader.m] designated initializer fails to return a non nil bundle for the GCDWebServers framework's bundle.

I believe it is related to an integration issue (maybe missing install path in GCDWedServers project)

Environemnt:
Xcode 7.1
Target iOS Version: 8.0

**Comment by davidandreoletti** [comment]
Release 3.3 fixes the problem (commit: a6746147132f68b85666d04aca79e79fbbfcd92b).

**Comment by swisspol** [comment]
Great!
